plugin-locationcomponent/build.gradle.kts

```diff
@@ -23,7 +23,7 @@ android {
 }
 
 dependencies {
-  api(Dependencies.mapboxAndroidCore)
```

Comment:
Need to remember to add this dependency back to avoid api breaking change, internally we are migrated to the common sdk compat package.

Reply:
Remove?

Reply:
@pengdev if we're about to merge it - let's uncomment all this code related to this dependency

Reply:
yes, this is done now, this api dependency is brought back.